ffmpeg

```diff
@@ -26,6 +26,9 @@ ffmpeg -i input_video.mp4 -acodec copy -vcodec copy -f mov output_video.mov
 # To convert mov to mp4
 ffmpeg -i input_video.mov -vcodec copy -acodec copy output_video.mp4
 
+# Change INPUT's FPS to 30 and save as OUTPUT (https://trac.ffmpeg.org/wiki/ChangingFrameRate)
+ffmpeg -i input_video.ext -filter:v fps=fps=30 output_video.ext
```

Comment:
```suggestion
# To change INPUT's FPS to 30 and save as OUTPUT (https://trac.ffmpeg.org/wiki/ChangingFrameRate):
ffmpeg -i <in-video.ext> -filter:v fps=fps=30 <out-video.ext>
```

Optional fix: `To` can be used for consistency.
Optional fix: trailing colon can be added.
Optional fix: placeholders for input and output files can be used.
Optional fix: [placeholder content can use `in-` and `out-` prefixes for input and output files respectively](https://github.com/cheat/cheatsheets/pull/194/files#diff-438478e7713ffdb96be895cd2cba03144f30c04f1442ae246acdeb2f1c13efd5R41).